### ISSUE TITLE
how to config apple DNS resolver,the envoy proxy doc is blank

### ISSUE THREAD

**Issue by 13567436138** [envoyproxy/envoy#20499]
**If you are reporting *any* crash or *any* potential security issue, *do not*
open an issue in this repo. Please report the issue via emailing
envoy-security@googlegroups.com where the issue will be triaged appropriately.**

*Title*: *One line description*
how to config apple DNS resolver,the envoy proxy doc is blank
*Description*:
>Describe the issue.
I want to config bootstrap typed_dns_resolver_config,use apple dns resolver,but the doc is blank
[optional *Relevant Links*:]
>Any extra documentation required to understand the issue.

**Comment by daixiang0** [comment]
Yes, it is blank, refer to https://github.com/envoyproxy/envoy/blob/0b769f6e299dac9115cba10008ce984d0ebf951a/api/envoy/extensions/network/dns_resolver/apple/v3/apple_dns_resolver.proto#L16.

Read cross the codes, seems we make default config for it, we really need docs for this,
